### ISSUE TITLE
Soundfont 404 not found

### ISSUE THREAD

**Issue by R-N** [Music-and-Culture-Technology-Lab/omnizart#84]
I think the soundfont drive link in the colab is dead? Where do I get this file?

Stack trace:
```
INFO:Vocal Transcription:Transcription finished
Downloading soundfont...
---------------------------------------------------------------------------
HTTPError                                 Traceback (most recent call last)
[<ipython-input-3-6a53ec2abbf5>](https://localhost:8080/#) in <module>
     52       "16RM-dWKcNtjpBoo7DFSONpplPEg5ruvO",
     53       file_length=31277462,
---> 54       save_name=SF2_FILE
     55     )
     56 

5 frames
[/usr/local/lib/python3.7/dist-packages/omnizart/remote.py](https://localhost:8080/#) in download_large_file_from_google_drive(url, file_length, save_path, save_name, unzip)
    176     cookie_jar = http.cookiejar.MozillaCookieJar()
    177     opener = urllib.request.build_opener(urllib.request.HTTPCookieProcessor(cookie_jar))
--> 178     resp = opener.open(url)
    179     cookie_jar.save("./.cookie")
    180     cookie = resp.getheader("Set-Cookie")

[/usr/lib/python3.7/urllib/request.py](https://localhost:8080/#) in open(self, fullurl, data, timeout)
    529         for processor in self.process_response.get(protocol, []):
    530             meth = getattr(processor, meth_name)
--> 531             response = meth(req, response)
    532 
    533         return response

[/usr/lib/python3.7/urllib/request.py](https://localhost:8080/#) in http_response(self, request, response)
    639         if not (200 <= code < 300):
    640             response = self.parent.error(
--> 641                 'http', request, response, code, msg, hdrs)
    642 
    643         return response

[/usr/lib/python3.7/urllib/request.py](https://localhost:8080/#) in error(self, proto, *args)
    567         if http_err:
    568             args = (dict, 'default', 'http_error_default') + orig_args
--> 569             return self._call_chain(*args)
    570 
    571 # XXX probably also want an abstract factory that knows when it makes

[/usr/lib/python3.7/urllib/request.py](https://localhost:8080/#) in _call_chain(self, chain, kind, meth_name, *args)
    501         for handler in handlers:
    502             func = getattr(handler, meth_name)
--> 503             result = func(*args)
    504             if result is not None:
    505                 return result

[/usr/lib/python3.7/urllib/request.py](https://localhost:8080/#) in http_error_default(self, req, fp, code, msg, hdrs)
    647 class HTTPDefaultErrorHandler(BaseHandler):
    648     def http_error_default(self, req, fp, code, msg, hdrs):
--> 649         raise HTTPError(req.full_url, code, msg, hdrs, fp)
    650 
    651 class HTTPRedirectHandler(BaseHandler):

HTTPError: HTTP Error 404: Not Found
```

**Comment by BreezeWhite** [comment]
I've uploaded the soundfont file and you can manually download [here](https://github.com/Music-and-Culture-Technology-Lab/omnizart/releases/download/checkpoints-20211001/soundfont.sf2).

**Comment by BreezeWhite** [comment]
Fixed in #85.
